File: .tmp_py4013/batch_08_dump/2883_drop_missing_data.js
// LeetCode 2883 - Drop Missing Data
// https://leetcode.com/problems/drop-missing-data/

/**
 * @param {object[]} students
 * @return {object[]}
 */
var dropMissingData = function(students) {
    return students.filter((r) => {
        const name = Array.isArray(r) ? r[1] : r.name;
        return name !== null && name !== undefined && name !== '';
    });
};

File: .tmp_rs4013/rust_check_12/2883_drop_missing_data.rs
struct Solution;
// LeetCode 2883 - Drop Missing Data
// https://leetcode.com/problems/drop-missing-data/

impl Solution {
    pub fn drop_missing_data(students: Vec<Vec<i32>>) -> Vec<Vec<i32>> {
        students
            .into_iter()
            .filter(|r| r.len() < 2 || r[1] != 0)
            .collect()
    }
}

fn main() {}
